### DISCUSSION TITLE
writing/2024/05/11/low-hanging-fruit-for-rag-search/

### DISCUSSION THREAD

**Discussion by giscus** [jxnl/blog#58] [General]
# writing/2024/05/11/low-hanging-fruit-for-rag-search/

Notes about my hobbies and other things I find interesting.

https://jxnl.co/writing/2024/05/11/low-hanging-fruit-for-rag-search/

**Comment by sudarshansivakumar** [comment]
To understand the 6th point more clearly, is the suggestion that we only embed possible question-answer pairs, or a combination of the text chunk and question answer pairs.
For instance assuming text chunk = _Machine learning (ML) is a field of study in artificial intelligence concerned with the development and study of statistical algorithms_ should the content that is embedded be  : 

1.  _What is Machine Learning?_ OR 
2.  _What is Machine Learning?_Machine learning (ML) is a field of study in artificial intelligence concerned with the development and study of statistical algorithms_ _
